editor/editor_dock_manager.cpp

```diff
@@ -149,17 +149,53 @@ void EditorDockManager::_update_layout() {
 	}
 	EditorNode::get_singleton()->edit_current();
 	dock_context_popup->docks_updated();
+	_update_docks_menu();
 	EditorNode::get_singleton()->save_editor_layout_delayed();
 }
 
+void EditorDockManager::_update_docks_menu() {
+	docks_menu->clear();
+	docks_menu->reset_size();
+
+	const Ref<Texture2D> icon = docks_menu->get_editor_theme_icon(SNAME("Window"));
+	const Color closed_icon_color_mod = Color(1, 1, 1, 0.5);
+
+	// Add docks.
+	docks_menu_docks.clear();
+	int id = 0;
+	for (const KeyValue<Control *, DockInfo> &dock : all_docks) {
+		if (dock.value.shortcut.is_valid()) {
+			docks_menu->add_shortcut(dock.value.shortcut, id);
+			docks_menu->set_item_text(id, dock.value.title);
+		} else {
+			docks_menu->add_item(dock.value.title, id);
+		}
+		docks_menu->set_item_icon(id, icon);
+		if (!dock.value.open) {
+			docks_menu->set_item_icon_modulate(id, closed_icon_color_mod);
+		}
+		docks_menu->set_item_disabled(id, !dock.value.enabled);
```

Comment:
Features that are disabled via Feature Profiles usually get **hidden** in the UI, not disabled. Seems like a simple change to skip adding the menu item in this case, instead of disabling.

Reply:
Sorry I forgot about this for a while. 
- Opened PR https://github.com/godotengine/godot/pull/99550